sellotape/sellotape_dj/sellotape_dj/auth_app/templates/home.html

```diff
@@ -0,0 +1,43 @@
+<!-- home.html -->
+
+    {% extends 'base.html' %}
+    {% block title %} Home {% endblock %}
+    {% block content %}
+      <div class="row">
+
+            <!-- Add from here -->
+        {% for ass in backends.associated %}
```

Comment:
I'm seeing what seems to be code repetition below, could you avoid that? 

Also `ass` is not a good variable name - it does not help me understand what it contains.

Reply:
Yes, you're right. In the development stages the variables were different for different providers, but it eventually changed and I didn't notice this. Updating in next commit.